cms/templates/content_libraries/xblock_iframe.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
```

Comment:
There are at least four versions of this iframe JS adapter file that I know of now:
* this
* [`frontend-app-library-authoring/src/library-authoring/edit-block/LibraryBlock/wrap.js`](https://github.com/edx/frontend-app-library-authoring/blob/e189348dba7ea086c3c223055f0054a7bfa2eb8c/src/library-authoring/edit-block/LibraryBlock/wrap.js)
* [Ramshackle's wrap.ts](https://github.com/open-craft/ramshackle/blob/master/ramshackle/js-src/Block/wrap.ts) (admittedly a temporary project)
* LabXchange

We don't need to solve this now, but I've been worried about how fragile this approach is and would love to come up with a better solution or some way to record these JS dependencies in a common place. Or at the very least we can declare this one or the frontend-app-library-authoring one as the "main" version and put comments in the others that they should be based on the "main" version?